sonar-text-dotnet/pipeline_scripts/run_uts.ps1

```diff
@@ -0,0 +1,4 @@
+$utsDllPath="$env:PROJECT_DIR\\src\\SonarLint.Secrets.DotNet.UnitTests\\bin\\Release\\net48\\SonarLint.Secrets.DotNet.UnitTests.dll"
+
+Write-Host "Running UTs from path: $utsDllPath"
+dotnet test  $utsDllPath  --configuration "Release"  --no-build
```

Comment:
```suggestion
dotnet test $utsDllPath --configuration "Release" --no-build
```

Can we maybe run `dotnet test` on the solution instead of dlls? In this way, we will not have to change the script if we decide to change the targeting version or to add more.